build/xbuild.include

```diff
@@ -3,17 +3,6 @@ XBUILD_ARGS=/verbosity:normal /nologo /property:CodePage=65001
 ALL_CSPROJ=$(wildcard *.csproj)
 MAIN_SLN=$(top_builddir)/F-Spot.sln
 
-#if ENABLE_MACPLATFORM
-#PROFILE_NAME=DebugMac
-#endif
-#if ENABLE_GNOMEPLATFORM
-#PROFILE_NAME=Debug
-#endif
-
-#if ENABLE_WINDOWSPLATFORM
-#PROFILE_NAME=DebugWin32
-#endif
-
```

Comment:
I removed it as this is the only occurrence of these profile names and they were already commented out. Didn't know about your plans to support mac and win.
Personally I like to prevent additional configurations in solutions and projects as they clutter up the project files. Also, configuring those configurations (selection of projects/configurations to build) is only possible via GUI and verifying it in a diff is almost impossible due to the GUIDs (lots of jumping around to see which project is active in which configuration). I prefer to configure builds via properties (environment or directly given to xbuild) and have the projects react accordingly (although that might not work in all cases...).